code/network/multi_interpolate.h

```diff
@@ -0,0 +1,208 @@
+#pragma once
+
+#include "globalincs/pstypes.h"
+#include "globalincs/systemvars.h"
+#include "io/timer.h"
+#include "math/vecmat.h"
+
+// A manually written copy of Int max to keep linux CI from complainint.
+constexpr int COMPAT_INT_MAX = 2147483647;
+
+struct physics_info;
+
+constexpr int DEFAULT_FRAME_TIME = 17; // 1000 ms / 60 fps 
```

Comment:
How dependent on 60 fps is this to work correctly?

Standalone defaults to 30 fps and that can only be changed in the UI. Plus the ultimate goal with the new standalone changes is to make it dynamic, 1 - 120 fps, so it can respond to packets as efficiently as possible.

Reply:
We really just need some kind of default.  On one of my tests, the client was missing about 20 early frames, and then wasn't missing one until about frame 700.  I can try to have it check if it is on a standalone and adjust?  But dropped packets are not as common a thing as one would expect.

Reply:
With the change to use a full timestamp and a multi bump now being required, this is no longer an issue.